apps/rx-store-website/docs/introduction/core-concepts.md

```diff
@@ -0,0 +1,57 @@
+---
+id: getting-started
+title: Getting Started with Rx Store
+---
+
+`Rx Store` is a reactive store for frontend UI libraries (such as React), making it easy for you to build your application's logic using [RxJs](https://rxjs-dev.firebaseapp.com/) streams. If you are not yet familiar with RxJs, [this video](https://www.youtube.com/watch?v=ewcoEYS85Co) is a great introduction!
```

Comment:
btw, rxjs has it's own domain now: https://rxjs.dev/ -- this one is linked from the repo header.

(while they seem to neglect proper google search update)